pkg/k8s/service.go

```diff
@@ -4,14 +4,13 @@
 package k8s
 
 import (
+	"context"
 	"fmt"
+	"github.com/sirupsen/logrus"
+	v1 "k8s.io/api/core/v1"
 	"net"
 	"net/url"
 	"strings"
-	"time"
-
-	"github.com/sirupsen/logrus"
-	v1 "k8s.io/api/core/v1"
 
 	"github.com/cilium/cilium/pkg/annotation"
 	"github.com/cilium/cilium/pkg/cidr"
```

Comment:
These lines shouldn't be changed.

Reply:
> These lines shouldn't be changed.

I have reverted. And the latest code has been commited.